checks/contributors.go

```diff
@@ -38,7 +38,9 @@ func Contributors(c checker.Checker) checker.CheckResult {
 
 	companies := map[string]struct{}{}
 	for _, contrib := range contribs {
-		if contrib.GetContributions() >= 5 {
+		const contributorsCount = 5
+		//nolint:nestif
```

Comment:
I included `//nolint:nestif` because the golangci was complaining about nested if. I didn't want to fix this in this PR. I am planning to do it in the coming PR's. Thanks